core/frontend/src/render/webgl/Contours.ts

```diff
@@ -0,0 +1,167 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module WebGL
+ */
+
+import { assert, dispose, Id64 } from "@itwin/core-bentley";
+import { ContourDisplay, PackedFeature, RenderFeatureTable } from "@itwin/core-common";
+import { WebGLDisposable } from "./Disposable";
+import { GL } from "./GL";
+import { UniformHandle } from "./UniformHandle";
+import { TextureUnit } from "./RenderFlags";
+import { System } from "./System";
+import { Target } from "./Target";
+import { Texture2DDataUpdater, Texture2DHandle, TextureHandle } from "./Texture";
+import { BatchOptions } from "../../common/render/BatchOptions";
+import { ContourUniforms } from "./ContourUniforms";
+
+function computeWidthAndHeight(nEntries: number, nRgbaPerEntry: number): { width: number, height: number } {
```

Comment:
Why isn't this just `computeDimensions(nEntries, nRgbaPerEntry, 0, System.instance.maxTextureSize)`?

Reply:
I was following pattern in FeatureOverrides.ts.  Looking now, I see that FeatureOverrides could do what you are suggesting.
However, I remember I had to change the code here to deal with the fractional nRgbaPerEntry.  I think it ended up just being the added ceil on line 23.
Also, neither this nor FeatureOverrides have the extra test and bump height at the end, but I am not sure how that could happen?

Reply:
Oh, I also changed line 25 from < to <=.

Reply:
Will it work to incorporate those two changes into computeDimensions and then replace this function and the one in FeatureOverrides with computeDimensions call?

Reply:
Okay, I went ahead and modified computeDimensions to add the ceil, changed < to <=, and removed the 2 redundant lines at the end.  Then I changed both FeatureOverrides.ts and Contours.ts to call computeDimensions instead of having their own functions.  I ran an ImageTests run with this and did not see any issues.

Reply:
@pmconne you didn't comment on my changes to computeDimensions, and now I see the AuxChannelTable test is failing (line 147) because it didn't expect it to fit in width 8 with only 1 row.  With my change, it allows it to use the full width (maxSize here) before going to square, so this is what I would expect.  But since the test was coded not to, I feel I should ask.  Is there some reason not to use the full max width, or should I just change the test?

Reply:
No, it's just a result of you swapping `<` and `<=`.